### ISSUE TITLE
Port WavPlayer Seed Example

### ISSUE THREAD

**Issue by andrewikenberry** [electro-smith/DaisyDuino#13]
https://github.com/electro-smith/DaisyExamples/tree/master/seed/WavPlayer

**Comment by andrewikenberry** [comment]
This actually needs to get flushed out in the CPP version as well.

**Comment by beserge** [comment]
This should probably be moved to the pod folder, no?

**Comment by andrewikenberry** [comment]
@beserge yes you are correct, sorry about that. It should be a pod example
